doc/releases/changelog-dev.md

```diff
@@ -343,6 +343,9 @@
 * `qml.transforms.map_batch_transform` is deprecated, since a transform can be applied directly to a batch of tapes.
   [(#5676)](https://github.com/PennyLaneAI/pennylane/pull/5676)
 
+* `split_non_commuting` returns the unmodified tape if there are no measurements, instead of an empty list.
+  [(#5869)](https://github.com/PennyLaneAI/pennylane/pull/5869)
```

Comment:
Since this bug didn't exist in the original `split_non_commuting`, I would simply add this PR as a follow-up to the entry for the unified `split_non_commuting` rather than as a bug fix.

Reply:
done